wrapanapi/systems/openstack.py

```diff
@@ -173,6 +173,17 @@ def all_ips(self):
         # raw.networks is dict of network: [ip, ip] key:value pairs
         return [ip for nets in self.raw.networks.values() for ip in nets]
 
+    @property
+    def floating_ip(self):
```

Comment:
Consider having explicit methods for `floating_ip` and `fixed_ip`.

Reply:
done